### PR TITLE
Add labels to all commands

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: replicated-gcommands/replicated-gcommands.plugin.zsh [hunk 1]
```diff
@@ -42,33 +44,33 @@ gcreate() {
...
   genv
-  local usage="Usage: gdelete [INSTANCE_NAME_PREFIX]"
-  local instance_name_prefix="$1"
-  if [ -z "$instance_name_prefix" ]; then echo "Must provide prefix"; echo "${usage}"; return; fi
-  if ! gcloud compute instances list --filter="labels.owner:$GUSER" | awk '{if(NR>1)print}' | grep -q "^$instance_name_prefix"; then echo "no instances match pattern \"^$instance_name_prefix\""; echo "${usage}" return 1; fi
-  gcloud compute instances delete --delete-disks=all $(gcloud compute instances list --filter="labels.owner:$GUSER" | awk '{if(NR>1)print}' | grep "^$instance_name_prefix" | awk '{print $1}' | xargs echo)
+  local usage="Usage: gdelete [INSTANCE_NAMES]"
+  local instance_name_prefix=""
+  if [ -n "$1" ]; then
+    instance_name_prefix="${GUSER}-$1"
```
Comment (chris-sanders) [inline replicated-gcommands/replicated-gcommands.plugin.zsh:50]: Using tags why are you inserting a hard coded prefix that users can't change? A user can get this same effect by simply setting a prefix so I tend to think the scripts shouldn't force it.

File: replicated-gcommands/replicated-gcommands.plugin.zsh [hunk 2]
```diff
@@ -42,33 +44,33 @@ gcreate() {
...
 
-# TOOD: Udate for owner labels
 gonline() {
   genv
   local usage="Usage: gonline [INSTANCE_NAMES]"
   if [ "$#" -lt 1 ]; then echo "${usage}"; return 1; fi
   local instance
   for instance in "$@"; do
-    local instance_name_prefix="${GPREFIX}${instance}"
+    local instance_name_prefix="${GUSER}-${instance}"
```
Comment (chris-sanders) [inline replicated-gcommands/replicated-gcommands.plugin.zsh:62]: As above

File: replicated-gcommands/replicated-gcommands.plugin.zsh [hunk 3]
```diff
@@ -78,21 +80,22 @@ gairgap() {
 gssh-forward() {
   # genv
   local usage="Usage: gssh-forward [INSTANCE_NAME]"
-  if [ "$#" -lt 1 ]; then echo "${usage}"; return 1; fi
-  local natip=$(gcloud compute instances describe $1 --format="value(networkInterfaces[0].accessConfigs[0].natIP)")
-  local ip=$(gcloud compute instances describe $1 --format="value(networkInterfaces[0].networkIP)")
-  # gcloud compute ssh --tunnel-through-iap $1 -- -L 8800:$address:8800 -L 8888:$address:8888
+  if [ "$#" -ne 1 ]; then echo "${usage}"; return 1; fi
+  local instance_name_prefix="${GUSER}-$1"
```
Comment (chris-sanders) [inline replicated-gcommands/replicated-gcommands.plugin.zsh:84]: as above

File: replicated-gcommands/replicated-gcommands.plugin.zsh [hunk 4]
```diff
@@ -102,33 +105,30 @@ gssh() {
...
 
-# TOOD: Udate for owner labels
 gattach() {
   genv
   local usage="Usage: gattach [INSTANCE_NAME] [DISK_NAME]"
   if [ "$#" -ne 2 ]; then echo "${usage}"; return 1; fi
-  local instance_name_prefix="${GPREFIX}$1"
-  local disk_name_prefix="${GPREFIX}disk-$2"
-  local device_name_prefix="${GPREFIX}$1-disk-$2"
+  local instance_name_prefix="${GUSER}-$1"
```
Comment (chris-sanders) [inline replicated-gcommands/replicated-gcommands.plugin.zsh:121]: as above I won't repeat it 3x times

File: replicated-gcommands/replicated-gcommands.plugin.zsh [hunk 5]
```diff
@@ -102,33 +105,30 @@ gssh() {
...
+  local disk_name_prefix="${GUSER}-disk-$2"
+  local device_name_prefix="${GUSER}-$1-disk-$2"
```
Comment (chris-sanders) [inline replicated-gcommands/replicated-gcommands.plugin.zsh:123]: I do think adding a prefix on the disks is fine, you can't (I don't think) label a disk and you don't really see the disk names either.

**Review by chris-sanders — CHANGES_REQUESTED** [review]
One particular comment what are your thoughts on that?

File: replicated-gcommands/replicated-gcommands.plugin.zsh (hunk 1 above)
Comment (emosbaugh) [reply replicated-gcommands/replicated-gcommands.plugin.zsh:50]: Makes sense. I'll update as a separate var.

**Comment by emosbaugh** [comment]
I've made GPREFIX a separate variable for adding a prefix to the name rather than repurposing GUSER.

**Pull request merged**